msbuild/Xamarin.MacDev.Tasks/Xamarin.MacDev.Tasks.csproj

```diff
@@ -78,6 +78,9 @@
     <Compile Include="..\..\tools\common\ObjCNameIndex.cs">
       <Link>external\ObjCNameIndex.cs</Link>
     </Compile>
+    <Compile Include="..\..\tools\common\Rewriter.cs">
+      <Link>external\ObjCNameIndex.cs</Link>
+    </Compile>
```

Comment:
Not sure why you need this if it's not being implemented as an MSBuild task anymore?